Formula/azure-cli.rb

```diff
@@ -3,10 +3,9 @@ class AzureCli < Formula
 
   desc "Microsoft Azure CLI 2.0"
   homepage "https://docs.microsoft.com/cli/azure/overview"
-  url "https://github.com/Azure/azure-cli/archive/azure-cli-2.17.0.tar.gz"
-  sha256 "0605059822bca1ac471afd236a432824c7675070d99ee40863801df3aa7cea54"
+  url "https://github.com/Azure/azure-cli/archive/azure-cli-2.17.1.tar.gz"
```

Comment:
Are we still not switching to the PyPI URL?

Note that this formula has a large number of outdated resource blocks, one of which caused a bug for the last version. Switching to PyPI will help automate the update of those resource blocks.

Reply:
It takes some time to switch to pypi as we need to support edge builds as well and verify the impact of version bump of the resources. The resources are pinned in the [requirements file](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/requirements.py3.Darwin.txt) now for stability.

Reply:
Well, the `cffi` version in your requirements file lead to this bug: https://github.com/Homebrew/brew/issues/10152#issuecomment-753334171

So I think you at least need to update that.

Reply:
Sure. I will submit a PR to update it.

Reply:
CC @Rylan12 re `azure-cli` resources

Reply:
@fengzhou-msft One clarification: by automatic, we mean that a maintainer/contributor doesn't have to spend time manually copying/pasting URLs and sha256s from the PyPI homepage. We still respect the pinned versions that are defined in `requirements.txt`. We can't do this check if we don't have a PyPI url, though, so I think it's actually more likely that resources will not be in sync with your project without the PyPI url.